controllers/topics-controller.js

```diff
@@ -29,3 +31,16 @@ exports.getArticles = (req, res, next) => {
     })
     .catch(next);
 };
+
+exports.getCommentsByArticleId = (req, res, next) => {
+  const { article_id } = req.params;
+  const promises = [fetchCommentsByArticleId(article_id)];
+  if (article_id) {
```

Comment:
There will always be an article_id if you hit this controller so this if statement is not needed.